tests/django_restframework_gis_tests/urls.py

```diff
@@ -22,6 +22,11 @@
         views.geojson_nullable_details,
         name='api_geojson_nullable_details',
     ),
+    path(
+        'geojson_nogeo/<int:pk>/',
+        views.geojson_nullable_details_nogeo,
+        name='api_geojson_nullable_details_nogeo',
+    ),
```

Comment:
This piece was entirely missing from the tests entirely, so when you did `reverse("api_geojson_location_details_nogeo")` in the tests it was impossible to resolve the URI to it.